audio/sonic/files/patch-sonic-mac.diff

```diff
@@ -0,0 +1,119 @@
+--- Makefile.orig	2020-06-30 13:32:50.000000000 +0700
++++ Makefile	2020-06-30 14:09:51.000000000 +0700
```

Comment:
I hope you're sharing all of these changes with the developers of sonic. If so, or if you got these changes from a patch developed by someone else, please mention at least the URL at the top of the patchfile.

Reply:
I will share back to the developer. The day I was working on this github was having problems (per their status page) and I couldn't file any issues or pull requests.